addons/website_event_sale/static/src/js/website_event_sale_ticket_details.js

```diff
@@ -0,0 +1,19 @@
+odoo.define('website_event_sale.ticket_details', function (require) {
```

Comment:
even if the parent class still use old odoo style import, can we use the new style here ?
```js
/** @odoo-module **/

import publicWidget from 'web.public.widget';

publicWidget.registry.ticketDetailsWidget.include({
    /**
     * Overriding the method to toggle the tickets registration
     * pricelist dropdown visibility on ticket details click
     */
    _onTicketDetailsClick: function(ev) {
        this._super(...arguments);
        if (this.foldedByDefault){
            $(ev.currentTarget).siblings('#o_wevent_tickets_pricelist').toggleClass('collapse');
        }
    }
});
```